.evergreen/config.yml

```diff
@@ -357,7 +357,10 @@ functions:
         script: |
           set -o xtrace
           ${PREPARE_SHELL}
-          PYTHON_BINARY=${PYTHON_BINARY} bash ${PROJECT_DIRECTORY}/.evergreen/run-mockupdb-tests.sh
+
+          cd ${PROJECT_DIRECTORY}
+          alias python=${PYTHON_BINARY}
```

Comment:
Is this `alias` needed or can we just do `$PYTHON_BINARY -m tox ...`?

Reply:
Oh so the alias is used within the tox config? Interesting.

Reply:
Yes, tox is configured to use whatever Python `python` points to so as long as we correctly set the path of `python` a host it should behave as expected.